tools/cmake/tests/multiple_executables/app1/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 Arm Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(APP_TARGET app1)
```

Comment:
Why define `APP_TARGET`? Seems like far less typing to just reuse `app1`, or `app2`. Is this part of some Mbed CMake style guide, or is mbed-os itself still relying on `APP_TARGET` somewhere?

Reply:
I'll change them to `app1`, `app2`.